docs/source/routing/graphos-features.mdx

```diff
@@ -1,187 +1,35 @@
 ---
 title: GraphOS Router Features
-subtitle: Use router features enabled by GraphOS plans
+subtitle: Features that requiring a licensed GraphOS plan 
 description: Unlock Enterprise features for the GraphOS Router by connecting it to Apollo GraphOS.
 redirectFrom: 
   - /router/enterprise-features
 ---
 
-A router connected to GraphOS, whether cloud-hosted or self-hosted, is called a **GraphOS Router**. It has access to specific GraphOS features depending on the connected GraphOS organization's plan. Refer to the [pricing page](https://www.apollographql.com/pricing#graphos-router) to compare GraphOS Router features across plan types.
+This page lists the additional features of GraphOS Router that are enabled via integration with a licensed GraphOS plan.
 
-## GraphOS Router features
+> Refer to the [pricing page](https://www.apollographql.com/pricing) to compare GraphOS Router features across plan types.
 
-The GraphOS Router supports a collection of premium features specific to GraphOS. These features include:
+## GraphOS Router licensed plan features
 
-- Real-time updates via [GraphQL subscriptions](/graphos/routing/operations/subscriptions)
-- Improved performance with [query batching](/graphos/routing/performance/query-batching)
 - Authentication of inbound requests via [JSON Web Token (JWT)](/graphos/routing/security/jwt)
 - [Authorization of specific fields and types](/graphos/routing/security/authorization) through the [`@requiresScopes`](/graphos/routing/security/authorization#requiresscopes), [`@authenticated`](/graphos/routing/security/authorization#authenticated), and [`@policy`](/graphos/routing/security/authorization#policy) directives
 - Incremental migration between subgraphs through [the progressive `@override` directive](/graphos/schema-design/federated-schemas/entities/migrate-fields#incremental-migration-with-progressive-override).
 - Redis-backed [distributed caching of query plans and persisted queries](/graphos/routing/performance/caching/distributed)
-- Redis-backed [entity caching of subgraph responses](/graphos/routing/performance/caching/entity)
```

Comment:
Curious about the omission of this--is this no longer a licensed feature?

Reply:
added back